tests/utils.py

```diff
@@ -17,8 +17,8 @@ def input_value(fun):
     return capturedOutput.getvalue()
 
 
-def assert_console(fun):
+def assert_console(fun, message: str):
     """Recebe função que printa algo na tela e realiza assert
     que verifica se foi printado."""
     unit = unittest.TestCase()
-    return unit.assertTrue(len(input_value(fun)) > 0)
+    return unit.assertEqual(message, input_value(fun))
```

Comment:
Essa função é utilizada em inúmeros testes do projeto, no qual, alguns não apresentam o parâmetro `message`, por isso os erros estão sendo apresentados no unittest pelo travis.

Sugiro fazer alguma verificação para checar o valor da mensagem.
Exemplo:
```python
if message:
    return unit.assertEqual(message, input_value(fun))
else:
    # Coloque algo aqui
    pass
```

Reply:
Muito obrigado, vou verificar essa condição, não me atentei à esse caso,

Eu fiz um CTRL+F de "assert_console" e pensei que tinha encontrado todos os usos.
valeu

Reply:
Lembre de dar um valor padrão para o parâmetro, dessa forma não vai ser apontado erro caso estejam chamando o `assert_console` e não estiverem passando o valor do `message`.

Reply:
> Lembre de dar um valor padrão para o parâmetro, dessa forma não vai ser apontado erro caso estejam chamando o `assert_console` e não estiverem passando o valor do `message`.
> 
> Acredito que dessa forma seja mais fácil e mais correto do que a que eu apontei acima.

O ideal é que cada teste tenha sua mensagem verificada mesmo. Mas tem umas mensagens gigantescas, para verificar elas daria bastante trabalho, talvez até necessitasse criar arquivos de texto com as mensagens para comparar, algo a ser discutido, por enquanto deixar um valor default como mencionado pelo Limão é o suficiente.